charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.9.5
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.53.10
+version: 5.54.0
```

Comment:
It's patch updates, so I think minor version bump is enough.

Reply:
done